.github/workflows/codeforafrica-deploy-review-app.yml

```
@@ -0,0 +1,124 @@
+name: Code for Africa | Deploy | REVIEW APP
+
+on:
+  pull_request:
+    paths:
+      - "apps/codeforafrica/**"
+      - "Dockerfile.codeforafrica"
+      - ".github"
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.ref }}"
+  cancel-in-progress: false
+
+env:
+  DOKKU_REMOTE_BRANCH: "master"
+  DOKKU_REMOTE_URL: "ssh://azureuser@ui-1.dev.codeforafrica.org"
+  IMAGE_NAME: "codeforafrica/codeforafrica-ui"
+  ORIGINAL_APP_NAME: "codeforafrica-ui"
+  NEXT_PUBLIC_APP_URL: "http://codeforafrica-ui-pr-${{github.event.pull_request.number}}.cms.review.codeforafrica.org"
```

```suggestion
  NEXT_PUBLIC_APP_URL: "http://codeforafrica-ui-pr-${{github.event.pull_request.number}}.dev.codeforafrica.org"
```

Can't this be accessible via `https`?